workflow-templates/python-build-and-lint.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: pipdeptree
```

Comment:
Doesn't let me apply comment to proper line, but line 22, the pip install shouldn't have package name included. That will install the latest deployed, not the latest developed which is what this should be building and linting.